diff --git a/py/selenium/webdriver/firefox/firefox_binary.py b/py/selenium/webdriver/firefox/firefox_binary.py
index 1bf072e..ab7f392 100644
--- a/py/selenium/webdriver/firefox/firefox_binary.py
+++ b/py/selenium/webdriver/firefox/firefox_binary.py
@@ -188,9 +188,12 @@ class FirefoxBinary(object):
             if not os.path.exists(library_path):
                 os.makedirs(library_path)
             import shutil
-            shutil.copy(os.path.join(os.path.dirname(__file__), path,
-              self.NO_FOCUS_LIBRARY_NAME),
-              library_path)
+            try:
+                shutil.copy(os.path.join(os.path.dirname(__file__), path,
+                    self.NO_FOCUS_LIBRARY_NAME),
+                    library_path)
+            except:
+                pass  # A Man's Gotta Do ...
             built_path += library_path + ":"
 
         return built_path
